### ISSUE TITLE
(

### ISSUE THREAD

**Issue by oleg-skliarenko** [avocode/avocode#546]
IMPORTANT: Please try to describe how did it happen.

-----
App v.2.4.5
Source UID: 115787
```
Error: invalid stored block lengths
    at Zlib._handle.onerror (zlib.js:345:17)
```

**Comment by jankuca** [comment]
Thank you for the report. We've been investigating this issue for a while. It's a duplicate of #447 which is why I'm going to close this one; we'll let you know when this is resolved there.

**Comment by petrbrzek** [comment]
This error should be now fixed with the latest Avocode 2.5.3. Please try it and let us know.
